knative-operator/deploy/operator.yaml

```diff
@@ -1,43 +0,0 @@
-apiVersion: apps/v1
```

Comment:
Why remove this? Seems like an unrelated change.

Reply:
It is part of the refactoring and it seems a relic of the past.